src/index.js

```diff
@@ -7,15 +7,17 @@ const resolvers = require("./resolvers");
 
 const server = new ApolloServer({
   typeDefs,
-
   resolvers,
   dataSources: () => ({
     earthAPI: new EarthAPI()
   }),
-
   //introspection flag fixes 400 response errors on heroku
-  introspection: true
-
+  introspection: true,
+  //add console error logs
+  formatError: err => {
+    console.log(err.stack);
+    return err;
+  }
 });
```

Comment:
We will have a merge conflict because I also changed this file (adding mocks: true)